resources-service/src/resource/resource-model.ts

```diff
@@ -82,52 +86,100 @@ export type ReferrableResource = {
 // The full resource & the search result are synonyms for now, but the full resource should grow to be a superset
 export type ReferrableResourceSearchResult = ReferrableResource;
 
-export const getResource = async (
-  accountSid: AccountSID,
-  resourceId: string,
-): Promise<ReferrableResource | null> => {
-  const record = await getById(accountSid, resourceId);
-  return record ? { ...record, attributes: attributeObjectGraphFromKeys(record.attributes) } : null;
-};
+export const resourceModel = (cloudSearchConfig: CloudSearchConfig) => {
+  const cloudSearchClient = resourceCloudSearchClient(cloudSearchConfig);
 
-export const searchResources = async (
```

Comment:
Sorry, moving the model to be an instantiated object really screwed up the diff...